libs/SalesforceSDK/src/com/salesforce/androidsdk/util/LogoutCompleteReceiver.kt

```diff
@@ -29,20 +29,45 @@ package com.salesforce.androidsdk.util
 import android.content.BroadcastReceiver
 import android.content.Context
 import android.content.Intent
+import com.salesforce.androidsdk.accounts.UserAccount
 import com.salesforce.androidsdk.app.SalesforceSDKManager
+import com.salesforce.androidsdk.app.SalesforceSDKManager.Companion.USER_ACCOUNT_KEY
 import com.salesforce.androidsdk.auth.OAuth2.LogoutReason
-import java.util.Locale
 
 /**
  * Listens for the logout complete event, and acts on it.
  */
 abstract class LogoutCompleteReceiver : BroadcastReceiver() {
     override fun onReceive(context: Context, intent: Intent) {
         if (intent.action == SalesforceSDKManager.LOGOUT_COMPLETE_INTENT_ACTION) {
-            val reason = intent.getStringExtra(SalesforceSDKManager.LOGOUT_REASON_KEY) ?: LogoutReason.UNKNOWN.toString()
-            onLogoutComplete(LogoutReason.valueOf(reason.uppercase(Locale.ROOT)))
+            val reason = intent.getStringExtra(SalesforceSDKManager.LOGOUT_REASON_KEY)?.let {
+                LogoutReason.valueOf(it.uppercase())
+            } ?: LogoutReason.UNKNOWN
+            val userAccount = intent.getBundleExtra(USER_ACCOUNT_KEY)?.let { bundle ->
+                UserAccount(bundle)
+            }
+            @Suppress("DEPRECATION")
+            onLogoutComplete(reason)
+            onLogoutComplete(reason, userAccount)
         }
     }
 
-    protected abstract fun onLogoutComplete(reason: LogoutReason)
+    /**
+     * Called when logout is complete.
+     *
+     * @param reason The reason for the logout. If no reason is available, [LogoutReason.UNKNOWN] is used.
+     */
+    @Deprecated(
+        message = "Use onLogoutComplete method that includes the userAccount that was logged out.",
+        replaceWith = ReplaceWith("onLogoutComplete(reason: LogoutReason, userAccount: UserAccount?)")
+    )
```

Comment:
NIT: We typically include the MSDK version we intend to remove the API in.  I would assume 14.0 in this case.